tests/migrations/helpers/test_update_legacy_filtered_positives_helper.py

```diff
@@ -153,3 +165,81 @@ def test_split_mongo_samples_by_version(unmigrated_mongo_testing_samples):
             assert samples == v2_unmigrated_samples
         else:
             raise AssertionError(f"Unexpected version '{version}'")
+
+
+# ----- test update_mlwh_filtered_positive_fields_batched method -----
+
+
+def update_mlwh_filtered_positive_fields_batched_batched_return_false_with_no_connection(config):
+    with patch("migrations.helpers.update_legacy_filtered_positives_helper.create_mysql_connection") as mock_connection:
+        version = "v2"
+        update_timestamp = datetime.now()
+
+        mock_connection().is_connected.return_value = False
+        result = update_mlwh_filtered_positive_fields_batched(config, [], version, update_timestamp)
+        assert result is False
+
+
+def test_update_mlwh_filtered_positive_fields_batched_raises_with_error_updating_mlwh(
+    config, mlwh_connection, migrated_mongo_testing_samples
+):
+    version = "v2"
+    update_timestamp = datetime.now()
+
+    with patch(
+        "migrations.helpers.update_legacy_filtered_positives_helper.run_mysql_execute_formatted_query",
+        side_effect=Exception("Boom!"),
+    ):
+        with pytest.raises(Exception):
+            update_mlwh_filtered_positive_fields_batched(
+                config, migrated_mongo_testing_samples, version, update_timestamp
+            )
+
+
+def test_update_mlwh_filtered_positive_fields_batched_calls_to_update_samples(
```

Comment:
This should be in the `db.py` as you are testing the result of the method you call in the `update_mlwh_filtered_positive_fields_batched` method.

I think here you should test that the `run_mysql_execute_formatted_query` is called with the expected parameters (ids, sql_arguments etc.)